config/np_network_internode/netperf_client_pod.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: mynetperfclientpod
+  namespace: kbench-pod-namespace
+  labels:
+    name: mynetperf
+    podtype: client
+spec:
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+      - labelSelector:
+          matchExpressions:
+          - key: podtype 
+            operator: In
+            values:
+            - server
+        topologyKey: kubernetes.io/hostname
+  hostname: netperfclient
+  subdomain: kbench-service-oid-0-tid-0
+  containers:
+  - image: gcr.io/google-containers/nginx
+    resources:
+      limits:
+        memory: "4200Mi"
+        cpu: "16000m"
+      requests:
+        memory: "4200Mi"
+        cpu: "1000m"
+        ephemeral-storage: "5Gi"
+    command: ["/bin/sh","-c"]
+    args: 
+      - sleep 2s;
+        apt-get update;
```

Comment:
We can remove these iperf/qperf related installations.

Reply:
Done.